src/components/UploadAvatar/uploadAvatar.css

```diff
@@ -1,11 +1,16 @@
-.upload_modal {
+.upload_avatar {
   display: flex;
   align-items: center;
   justify-content: center;
 }
 
+.upload_avatar .upload_header {
+  border-bottom: none !important;
```

Comment:
it's is not recommended to use !important. rather increase specificity of the selector by adding parent selectors